opencraft/settings.py

```diff
@@ -295,6 +295,10 @@
 # for provisioning Open edX app servers.
 DEFAULT_CONFIGURATION_VERSION = env('DEFAULT_CONFIGURATION_VERSION', default=DEFAULT_OPENEDX_RELEASE)
 
+# Git tag for latest Open edX release. Used as a default refspec for
+# configuration, edx-platform, forum, notifier, xqueue, and certs when creating production instances.
+LATEST_OPENEDX_RELEASE = env('LATEST_OPENEDX_RELEASE', default='named-release/dogwood.rc')
```

Comment:
@antoviaque I just realised that this might be problematic, since we are using the same version for forum, xqueue, certs etc. as well, so we would have to fork all of them and create opencraft-specific branches.  Then it might be better to use the standard branchname `named-release/dogwood.rc` and add additional commits to it on our fork.  (I was worried that it might be confusing if the release branch on our branch is ahead of upstream, but I don't think it's too bad, and the alternative of using a new branch name seems worse.)